components/coachindicator/stories/template.js

```diff
@@ -0,0 +1,27 @@
+import { html } from "lit-html";
+import { classMap } from "lit-html/directives/class-map.js";
+
+import "../index.css";
+
+export const Template = ({
+	rootClass = "spectrum-CoachMark",
+	isQuiet = false,
+	withPopover = false,
+	variant,
+	...globals
+}) => {
+	return html`
+		<div
+			class=${classMap({
+				[`${rootClass}Indicator`]: true,
+				[`${rootClass}Indicator--quiet`]: isQuiet,
+				[`${rootClass}Indicator--${variant}`]: variant !== "default",
+			})}
+			style="display: inline-block;vertical-align: top;"
```

Comment:
Is this inline style needed here? I'm seeing it display the same with or without it.

Reply:
no they aren't needed, this is a remnant of when they were displayed together